File: ctf-platform-web/src/api-services/models/createTeamReq.ts
// req.ts
export interface CreateTeamReq {
    /**
     * 战队名称（必填）
     */
    teamName: string;

    /**
     * 战队图标
     */
    teamIcon?: string;

    /**
     * 战队宣言
     */
    declaration?: string;

    /**
     * 战队介绍
     */
    teamIntroduction?: string;

    /**
     * 成立时间
     */
    establishmentTime: Date;

    /**
     * 战队邮箱
     */
    teamEmail?: string;

    /**
     * 战队网站
     */
    teamWebsite?: string;

    /**
     * 国家
     */
    country?: string;

    /**
   * 城市
   */
    city?: string;

    /**
     * 是否公开
     */
    isPublic: boolean;
}

File: CTFPlatForm/CTFPlatForm.Core/Dto/Team/CreateTeamReq.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CTFPlatForm.Core.Dto.Team
{
    public class CreateTeamReq
    {
        [Required(ErrorMessage = "战队名称缺失")]
        public string TeamName { get; set; }

        public string TeamIcon { get; set; }

        public string? Declaration { get; set; }

        public string? TeamIntroduction { get; set; }

        public DateTime? EstablishmentTime { get; set; }

        public string? TeamEmail { get; set; }

        public string? TeamWebsite { get; set; }

        public string? Country { get; set; }

        public string? City { get; set; }
        
        public bool IsPublic { get; set; }
        
    }
}
